src/fragments/lib/project-setup/flutter/platform-setup/ios.mdx

```diff
@@ -0,0 +1,15 @@
+Amplify requires a minimum deployment target of 13.0 and Xcode 13.2 or higher when targeting iOS.
+
+From your project root, navigate to the `ios/` directory and open the `Podfile` in a text editor of your choice. At the top of the file, update the target iOS platform to 13.0 or higher.
+
+```bash
+platform :ios, '13.0'
+```
+
+Open your project in Xcode and select Runner, Targets -> Runner and then the "General" tab. Under the "Minimum Deployments" section, update the iOS version to 13.0 or higher.
```

Comment:
Need to callout to open `.workspace` file? ditto macos

Reply:
Nah can be either